src/components/helpers/dom_helpers.ts

```diff
@@ -41,3 +41,29 @@ export function* iterateChildren(el: Node): Generator<Node> {
 export function getOpenedMenus(): HTMLElement[] {
   return Array.from(document.querySelectorAll<HTMLElement>(".o-spreadsheet .o-menu"));
 }
+
+const letterRegex = /^[a-zA-Z]$/;
+
+/**
+ * Transform a keyboard event into a shortcut string that represent this event. The letters keys will be uppercased.
```

Comment:
what's the rationale of uppercasing the letter keys but not the rest?

Reply:
Readability. Stuff like "PageDown" or "ArrowUp" is more readable in CamelCase than in all uppercase IMO.